lib/innit-client/tests/integration_test/mod.rs

```diff
@@ -31,10 +35,20 @@ mod integration_tests {
         client_key: String,
     }
 
+    const ENV_VAR_LOCALSTACK_URL: &str = "SI_TEST_LOCALSTACK_URL";
+    static INIT: Once = Once::new();
+
+    fn initialize_crypto() {
+        INIT.call_once(|| {
+            rustls::crypto::ring::default_provider()
+                .install_default()
+                .expect("Failed to install rustls crypto provider");
+        });
+    }
```

Comment:
What's going on here?

Reply:
The tests generate certs to ensure the certificate-based auth works. This, apparently, requires a crypto provider. However, you can only init one of these, so we poop into the once here.